libvast/CMakeLists.txt

```diff
@@ -255,20 +255,20 @@ string(APPEND VAST_FIND_DEPENDENCY_LIST "\nfind_package(spdlog 1.5.0 REQUIRED)")
 target_link_libraries(libvast PUBLIC spdlog::spdlog)
 dependency_summary("spdlog" spdlog::spdlog "Dependencies")
 
-# Link against simdjson.
-find_package(
-  simdjson
-  REQUIRED
-  # simdjson 0.8.0 exports it package configuration file under the wrong name,
-  # so we look for "simdjson-targets.cmake" as well.
-  CONFIGS
-  "simdjson-config.cmake"
-  "simdjson-targets.cmake")
-string(APPEND VAST_FIND_DEPENDENCY_LIST
-       "\nfind_package(simdjson REQUIRED CONFIGS "
-       "\"simdjson-config.cmake\" \"simdjson-targets.cmake\")")
+# Link against simdjson
```

Comment:
We should have an option `VAST_ENABLE_BUNDLED_SIMDJSON` that defaults to off.
Use the old code the option is disabled and the new one otherwise.